core/src/net/mod.rs

```diff
@@ -54,6 +55,28 @@ impl From<Transport> for Protocol {
     }
 }
 
+/// Session identifier, part of the `NetMessage` struct. Managed by Kompact internally, may be read
+/// by users to detect session loss, indicated by different SessionId's of NetMessages.
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct SessionId(Uuid);
```

Comment:
Oh, I overlooked this before: Maybe we should **not** derive `PartialOrd` and `Ord` here, since we are specifically not promising session id ordering as discussed before. I feel like giving people the ability to do `id_a < id_b` would suggest that this a sensible operation, when it really is not.

Reply:
Good catch, fully agree, leaving it there after changing to uuid was not intentional